.travis/docker-compose-travis.yml

```diff
@@ -3,8 +3,21 @@ version: "2"
 
 services:
   elasticsearch:
-    image: elasticsearch:1.5.2
+    image: elasticsearch:7.8.0
     container_name: "es.edx"
+    environment:
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
```

Comment:
For what it's worth, the container I'm using from devstack doesn't include these entire entries:

- ulimits
- ports

And seems to work fine.  Are these particularly necessary for travis or something?

Reply:
Done. It is not necessary